Dockerfile

```diff
@@ -27,13 +29,8 @@ RUN \
         python3-dev \
         python3-pip \
     && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
```

Comment:
> \-    apt-get clean && \
\-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \

Is it ok to remove this lines completely?

Reply:
You can find the same lines in the cleanup section around `apt-get autoremove`